src/current/_includes/v22.2/ui/transactions-filter.md

```diff
@@ -10,14 +10,21 @@
 
 The transaction fingerprints returned are determined by the selected **Search Criteria**:
 
-1. By default, the **Top** `100` transaction fingerprints **By** `% of All Runtime` are returned.
-  - To change the number returned, select `25`, `50`, or `500` from the dropdown.
-  - To change the column filter, select `Contention Time`, `Execution Count`, or `Transaction Time` from the dropdown.
+### Search Criteria ###
+
+By default, the **Top** `100` transaction fingerprints **By** `Transaction Time` for the `Past Hour` are returned.
+
+1. To change the number of results returned, select `25`, `50`, `100`, or `500` from the **Top** dropdown.
```

Comment:
on 22.2 we also have options 1000, 5000 and 10000

Reply:
to clarify: they're not under a "More" section like 23.1, this are just part of the regular list (same thing for the other top and limit comments I made for 22.2)